cmd/server/http/http.go

```diff
@@ -561,3 +563,27 @@ func getGrafanaVarForEnv(env, dev, staging, prod string) string {
 	}
 	return ""
 }
+
+func notifyPolicyFailure(email, title, errorMessage, token string, userMappings map[string]string) {
+	client, err := slack.NewClient(token)
+	if err != nil {
+		log.Errorf("error initializing Slack Client in notifyPolicyFailure: %v", err)
+	}
+
+	if !flow.IsLunarWayEmail(email) {
+		//check UserMappings
+		lwEmail, ok := userMappings[email]
+		if !ok {
+			log.Errorf("%s is not a Lunar Way email and no mapping exist", email)
+		}
+		email = lwEmail
+	}
+	slackId, err := client.GetSlackIdByEmail(email)
+	if err != nil {
+		log.Errorf("error obtaining slackId in notifyPolicyFailure: %v", err)
```

Comment:
You need to return after this line to avoid continuing.

```suggestion
		log.Errorf("error obtaining slackId in notifyPolicyFailure: %v", err)
		return
```